compiler/noirc_frontend/src/hir/type_check/mod.rs

```diff
@@ -479,7 +479,7 @@ mod test {
         expected_number_errors: usize,
         func_namespace: Vec<String>,
     ) {
-        let (program, errors) = parse_program(src);
+        let (program, errors) = parse_program(Default::default(), src);
```

Comment:
If `SrcId: Default` is only used for testing we could have a test-only function instead:
```rs
impl SrcId {
    #[cfg(test)]
    fn no_source() -> Self { ... }
}
```

Reply:
provided no_source function